lib/Twig/Loader/Filesystem.php

```diff
@@ -19,9 +19,9 @@ class Twig_Loader_Filesystem implements Twig_LoaderInterface
     /** Identifier of the main namespace. */
     const MAIN_NAMESPACE = '__main__';
 
-    protected $paths = array();
-    protected $cache = array();
-    protected $errorCache = array();
+    private $paths = array();
+    private $cache = array();
+    private $errorCache = array();
```

Comment:
That breaks Symfony which expects those properties to be available, but that's messy. We need a better abstraction.